3741/1/lib/gem_data_reader.rb

```diff
@@ -0,0 +1,68 @@
+require 'nokogiri'
+require 'json'
+require_relative '../lib/gem_data'
+
+# parsing data from web page of gem
+class GemDataReader
+  attr_reader :html
+
+  def initialize(gem_name)
+    @gem_name = gem_name
```

Comment:
always create
attr_reader for all incoming params and outgoing params:

```
attr_reader :gem_name
```

rename @gem_name -> gem_name